js/src/framework.ts

```diff
@@ -486,21 +485,26 @@ export async function runEvaluator(
                     return null;
                   }
 
-                  const scoreValues = Array.isArray(scoreValue)
+                  if (Array.isArray(scoreValue)) {
+                    for (const s of scoreValue) {
+                      if (!(typeof s === "object" && !isEmpty(s))) {
+                        throw new Error(
+                          `When returning an array of scores, each score must be a non-empty object. Got: ${s}`
```

Comment:
Technically passing an empty object will print out `[Object object]`. I wonder if we should print `JSON.stringify(s)`

Reply:
good idea